src/include/cpu/power/istep_18.h

```diff
@@ -0,0 +1,634 @@
+// [30] RWX REG_0X29_DATA_PARITY_ERROR_MASK: Error mask: RX-TType control register: data parity error.
+// [31] RWX REG_0X30_0X31_0X32_0X33_DATA_PARITY_ERROR_MASK: Error mask: data parity error on one of the following error handling registers:
+//      Error register (0x30)
+//      Error inject register (0x31)
+//      Error mask register (0x32)
+//      Error interrupt mask register (0x33)
+// [32] RWX REG_0X10_DATA_PARITY_ERROR_MASK: Error mask: chip control register (0x10): data parity error.
+// [33] RWX I_PATH_SYNC_CHECK_ERROR_MASK: Error mask: internal path: SYNC check.
+// [34] RWX I_PATH_FSM_STATE_PARITY_ERROR_MASK: Error mask: internal path: FSM state parity error.
+// [35] RWX I_PATH_TIME_REG_PARITY_ERROR_MASK: Error mask: internal path: time register: parity error.
+// [36] RWX I_PATH_TIME_REG_OVERFLOW_MASK: Error mask: internal path: time register overflow.
+// [37] RWX WOF_LOW_ORDER_STEP_COUNTER_PARITY_ERROR_MASK: Error mask: WOF counter or Low- Order-Step counter: parity error.
+// [38] RWX RX_TTYPE_0_MASK: Error mask: received TType-0.
+// [39] RWX RX_TTYPE_1_MASK: Error mask: received TType-1.
+// [40] RWX RX_TTYPE_2_MASK: Error mask: received TType-2.
+// [41] RWX RX_TTYPE_3_MASK: Error mask: received TType-3.
+// [42] RWX RX_TTYPE_4_MASK: Error mask: received TType-4.
+// [43] RWX RX_TTYPE_5_MASK: Error mask: received TType-5 when FSM is in running state.
+// [44] RWX PIB_SLAVE_ADDR_INVALID_ERROR_MASK: Error mask: PIB slave: invalid address.
+// [45] RWX PIB_SLAVE_WRITE_INVALID_ERROR_MASK: Error mask: PIB slave: invalid write.
+// [46] RWX PIB_SLAVE_READ_INVALID_ERROR_MASK: Error mask: PIB slave: invalid read.
+// [47] RWX PIB_SLAVE_ADDR_PARITY_ERROR_MASK: Error mask: PIB slave: Address parity error.
+// [48] RWX PIB_SLAVE_DATA_PARITY_ERROR_MASK: Error mask: PIB slave: data parity error.
+// [49] RWX REG_0X27_DATA_PARITY_ERROR_MASK: Error mask: TType control register(0x27): data parity error.
+// [50:52] RWX PIB_MASTER_RSP_INFO_ERROR_MASK: Error mask: PIB master: response infoerror.
+// [53] RWX RX_TTYPE_INVALID_ERROR_MASK: Error mask: received invalid TType from register 0x21.
+// [54] RWX RX_TTYPE_4_DATA_PARITY_ERROR_MASK: Error mask: data parity error on received TType-4 from register 0x21.
+// [55] RWX PIB_MASTER_REQUEST_ERROR_MASK: Error mask: PIB master; request while busy error.
+// [56] RWX PIB_RESET_DURING_PIB_ACCESS_ERROR_MASK: Error mask: PIB reset received during a PIB master or PIB slave operation.
+// [57] RWX EXTERNAL_XSTOP_ERROR_MASK: Error mask: TOD received an external checkstop.
+// [58] RWX SPARE_ERROR_MASK_58: Error mask: spare.
+// [59] RWX SPARE_ERROR_MASK_59: Error mask: spare.
+// [60] RWX SPARE_ERROR_MASK_60: Error mask: spare.
+// [61] RWX SPARE_ERROR_MASK_61: Error mask: spare.
+// [62] RWX OSCSWITCH_INTERRUPT_MASK: Error mask: Interrupt from the oscillator switch.
+// [63] RWX SPARE_ERROR_MASK_63: Error mask: spare.
```

Comment:
Those are masks for the same bits as in the previous register, there is no need to describe every bit. One sentence that these correspond to the above should be enough.

Reply:
Removed most of the comments and left only information about used parts